scripts/nmt/nmt.py

```diff
@@ -78,6 +78,7 @@ def parse_args() -> argparse.Namespace:
     )
     inputs.add_argument("--text-file", type=str, help="Path to file for translation")
     parser.add_argument("--dnt-phrases-file", type=str, help="Path to file which contains dnt phrases and custom translations")
+    parser.add_argument("--max-len-variation", type=str, help="Length variation limit used to control the allowed difference between source and translated text.")
```

Comment:
help comment change:
`Parameter to control the maximum variation between the length of source and translated text in terms of words`